partiql-planner/src/main/kotlin/org/partiql/planner/internal/typer/PlanTyper.kt

```diff
@@ -769,7 +769,7 @@ internal class PlanTyper(private val env: Env) {
 
         override fun visitRexOpCastUnresolved(node: Rex.Op.Cast.Unresolved, ctx: CompilerType?): Rex {
             val arg = visitRex(node.arg, null)
-            val cast = env.resolveCast(arg, node.target) ?: return ProblemGenerator.errorRex(
+            val cast = env.resolveCast(arg, node.target) ?: return ProblemGenerator.missingRex(
```

Comment:
Why MISSING if we _know_ the cast will never succeed?

Reply:
There are many conformance tests that follow this behavior. Though, in this PR I leave it up to the signaling mode.